### PR TITLE
add info about updating docsy theme

### PR DESCRIPTION
**Pull request by kbhawkey**
Add more information about updating the Docsy theme submodule.
See issue #22224 .

### CODE REVIEW

**Comment by kbhawkey** [comment]
@remyleone, let me know if this is not clear.

File: README.md [hunk 1]
```diff
@@ -8,12 +8,27 @@ This repository contains the assets required to build the [Kubernetes website an
 
 See the [official Hugo documentation](https://gohugo.io/getting-started/installing/) for Hugo installation instructions. Make sure to install the Hugo extended version specified by the `HUGO_VERSION` environment variable in the [`netlify.toml`](netlify.toml#L10) file.
 
-To run the website locally when you have Hugo installed:
+Before building the site, clone the Kubernetes website repository:
 
 ```bash
 git clone https://github.com/kubernetes/website.git
 cd website
 git submodule update --init --recursive
```
Comment (sftim) [inline README.md:16]:
```suggestion
git submodule update --init --recursive --depth 1
```

(to match the command below)
?

Comment (kbhawkey) [reply README.md:16]: Yes, I thought about it. Is there any situation when this is limiting?

Comment (sftim) [reply README.md]: I think it's fine, we're not expecting to ask contributors to hack on Docsy and its dependencies.

**Comment by kbhawkey** [comment]
Additional page previews:
https://deploy-preview-22225--kubernetes-io-master-staging.netlify.app/docs/contribute/new-content/new-content/#create-a-local-clone-and-set-the-upstream
https://deploy-preview-22225--kubernetes-io-master-staging.netlify.app/docs/contribute/new-content/new-content/#preview-locally

**Pull request merged**